cscs-checks/apps/quantumespresso/quantumespresso_check.py

```diff
@@ -110,7 +108,7 @@ def __init__(self, scale, variant):
         energy = sn.extractsingle(r'!\s+total energy\s+=\s+(?P<energy>\S+) Ry',
                                   self.stdout, 'energy', float)
         energy_diff = sn.abs(energy-energy_reference)
-        self.sanity_patterns = sn.all([
+        self.sanity_patterns += sn.all([
```

Comment:
Same comment here.

Reply:
Thanks @vkarak, I have changed the code accordingly